ClassicLootManager/Modules/Rosters/RosterManager/ItemValueCalculator.lua

```diff
@@ -43,6 +54,13 @@ local calculators = {
     [CONSTANTS.ITEM_VALUE_EQUATION.WOWPEDIA] = (function(ilvl, quality, multiplier, expvar, slot_multiplier)
         return math.pow(getItemValue(quality, ilvl), expvar) * multiplier * slot_multiplier
     end),
+    [CONSTANTS.ITEM_VALUE_EQUATION.CUSTOM] = (function(ilvl, quality, _, _, slot_multiplier, custom_expression)
+        return ExpressionParser:Evaluate(custom_expression, {
```

Comment:
Here you should guarantee the order of variables is always valid instead of sorting it inside every time